MoonCrystal/MoonCrystal/DeletedTotalCapacityView/MyFavoriteIdolCardView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  MyFavoriteIdolCardView.swift
+//  MoonCrystal
+//
+//  Created by sungkug_apple_developer_ac on 8/1/24.
+//
+
+import SwiftUI
+
+struct MyFavoriteIdolCardView : View {
+    @State var deletedTotalCapacity: String = "72GB"
+    @State var currentDeletedCapacity = "12GB"
+    @State var isEndView = false
+    
+    var userProfile: UserProfile?
+    
+    var body: some View {
+        ZStack {
+            cardBase
+                .frame(width: 307, height:  418)
+                .foregroundStyle(.gray900)
+            
+            VStack(spacing: 0) {
+                Spacer()
+                
+                Divider()
+                    .frame(width: 307)
+                    .frame(minHeight: 1)
+                    .overlay(.gray700)
+                
+                cardDescription
+                    .padding(.leading, 22)
+                    .padding(.vertical, 18)
+            }
+            .frame(width: 307, height:  418)
+            
+            if isEndView {
+                Capsule()
+                    .frame(width: 92,height: 40)
+                    .foregroundStyle(.pink100)
+                    .overlay(
+                        Text("+\(currentDeletedCapacity)")
+                            .font(.system(size: 16, weight: .bold))
+                            .foregroundStyle(.gray900)
+                    )
+                    .offset(x: 120, y: -209)
+            }
+        }
+    }
+    
+    var cardBase: some View {
+        RoundedRectangle(cornerRadius: 20)
+            .overlay(
+                Group {
+                    if let userProfile = userProfile, let uiImage = UIImage(data: userProfile.image) {
```

Comment:
같은 변수명으로 벗길 때는 이렇게 써도 가능해요!
```suggestion
                    if let userProfile, let uiImage = UIImage(data: userProfile.image) {
```

Reply:
와웅 좋은 정보 감사합니다 바로 적용하겠습니다!